packages/api/src/store/webhook-table.ts

```diff
@@ -29,7 +29,7 @@ export default class WebhookTable extends Table<DBWebhook> {
     const query = [sql`data->>'userId' = ${userId}`];
     if (streamId) {
       query.push(
-        sql`data->>'streamId' = ${streamId} OR data->>'streamId' IS NULL`
+        sql`(data->>'streamId' = ${streamId} OR data->>'streamId' IS NULL)`
```

Comment:
Nice! At some point we should just do that inside that `find()` function that concats everything with an `AND`. Shouldn't be the consumer's responsibility to care about this.